typespec/package-lock.json

```diff
@@ -9,15 +9,15 @@
       "version": "0.1.0",
       "license": "Apache-2.0",
       "dependencies": {
-        "@azure-tools/typespec-autorest": "latest",
-        "@azure-tools/typespec-azure-core": "latest",
-        "@azure-tools/typespec-azure-resource-manager": "latest",
-        "@azure-tools/typespec-providerhub": "latest",
-        "@typespec/compiler": "latest",
-        "@typespec/http": "latest",
-        "@typespec/openapi": "latest",
-        "@typespec/rest": "latest",
-        "@typespec/versioning": "latest"
```

Comment:
It looks like a change in behavior on depdendabot's part to prefer `*` over `latest`. 

@vishwahiremat - do you know the history here -- why we're using `latest` rather than specific version ranges?

Reply:
If anyone else knows, please chime in as well 👍

Reply:
no sure on why "latest" was added version here, i think this was the change added when we moved from cadl to typespec. But yeah I agree with using specific version for these.

Reply:
This is `package-lock.json` by the way.